app/dashboard/views/alias_contact_manager.py

```diff
@@ -44,6 +62,49 @@ def _check(form, field):
     return _check
 
 
+def create_contact(
+    user: User, alias: Alias, contact_address: str, fail_if_already_exist: bool = True
```

Comment:
Do you know if the `fail_if_already_exist` is used by caller?

Reply:
Not used. I'll remove it. Callers react to the exceptions triggered.